website/blog/2020-05-04-lithium.md

```diff
@@ -0,0 +1,312 @@
+
+For full details: https://github.com/scalameta/metals/milestone/13?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Scala 2.13.2 and basic Scala 3 support
+- improvements to Bloop integration
+- automatically add '+' on newline inside a string
+- code action "Import all missing symbols"
+
+## Basic Scala 3 support
+
+For a couple of last months we have been working on making Metals work with the
+Dotty compiler, which will become the new Scala 3. This effort included
+contributions to Metals, Bloop as well as to Dotty. Thanks to a great help from
+the Dotty compiler team, we now have an option to generate semanticDB files used
+for our internal indexes, which allowed some features to work without any
+changes to the core code. It is also important to mention that without
+[jvican](https://github.com/jvican) and his Bloop project, which already
+supports Scala 3, all this would have taken a much longer time.
+
+That said, there are some issues that are lacking in the Scala 3 support that
+are available for 2.x versions. Most notably, we are still missing all of the
+more advanced completions, filtering and sorting of completion items or auto
+imports. There are also some possible issues connected to optional braces and
+top level methods features. In the upcoming months we are planning to address
+most of those issues and implement the lacking features.
+
+You can find missing features in the
+[feature requests repository](https://github.com/scalameta/metals-feature-requests/issues?q=is%3Aissue+is%3Aopen+label%3Ascala3)
+and currently known bugs in the
+[main Metals repo.](https://github.com/scalameta/metals/issues?q=is%3Aissue+is%3Aopen+label%3A%22Scala+3%22)
+
+The first supported versions will be `0.23.0` and `0.24.0-RC1`, which are the
+latest stable and latests release candidate respectably. We plan to always
+support such two versions until there is a more stable release cycle for
+Scala 3.
+
+Please do report any issues you encounter, especially connected to some new
+Scala 3 features, which will be an immense help to the team.
+
+## Improvements related to Bloop/Metals integration
+
+There has been multiple reported issues in the last weeks about hanging
+compilation, which could make the Metals workspace hard to use. Our top priority
+is currently figuring out all the possible issues both in Metals as well in
+Bloop. Thanks to some fixes done by [jvican](https://github.com/jvican) we are
+hopeful that some of the issues might become less frequent. Those fixes are
+available in the current Bloop snapshots and in the version included in Metals
```

Comment:
```suggestion
There have been multiple reported issues in the last weeks about hanging
compilation, which can make the Metals workspace hard to use. Our top priority
has been figuring out all the possible issues both in Metals as well as in
Bloop. Thanks to some fixes done by [jvican](https://github.com/jvican), and some
work on the Metals side on the Bloop integration we're hopeful that some of the
issues are becoming less frequent. These fixes are included in this release.
```
I think we should re-word this a bit to sound a bit more positive. What do you think of this? I also cut out some of the details that are sort of imo irrelevant to this release doc like "these fixes are available in the current Bloop snapshots. I think this is a bit more positive and a bit more to the point.

Reply:
Thanks, I applied these changes.